src/code/__osMalloc.c

```diff
@@ -645,7 +645,7 @@ void* __osRealloc(Arena* arena, void* ptr, u32 newSize) {
     return ptr;
 }
 
-void* __osReallocDebug(Arena* arena, void* ptr, u32 newSize, const char* file, s32 line) {
+void* __osReallocDebug(Arena* arena, void* ptr, size_t newSize, const char* file, s32 line) {
     return __osRealloc(arena, ptr, newSize);
 }
 
```

Comment:
(I don't know where to put this comment, so I'll choose a semi arbitrary place).
Why you didn't change `ArenaImpl_GetSizes` to take `size_t*`?

Reply:
This comment also applies to wrappers of  `ArenaImpl_GetSizes`